lib/slack-ruby-bot/hooks/hello.rb

```diff
@@ -1,9 +1,13 @@
 module SlackRubyBot
   module Hooks
-    module Hello
-      extend Base
+    class Hello
+      attr_accessor :logger
 
-      def hello(client, _data)
+      def initialize(logger)
```

Comment:
well, the initialiser is not part of the interface, this particular hook takes the logger (I didn't want to touch any of the existing implementations) but it's 100% irrelevant if a hook is an object / class / lambda / whatever, so it's really irrelevant how it gets initialised

Not sure if you follow me :)